openlibrary/templates/lib/nav_head.html

```diff
@@ -129,7 +129,7 @@
           <li><a href="$homepath()/account">$_("Settings")</a></li>
           <li>
             <form name="logout" action="/account/logout" method="post">
-              <a href="#" onclick="document.forms['logout'].submit()">$_("Log out")</a>
+              <a href="#" onclick="document.forms['logout'].trigger('submit')">$_("Log out")</a>
```

Comment:
```suggestion
              <a href="#" onclick="document.forms['logout'].submit()">$_("Log out")</a>
```

`document.forms.logout` is not a jQuery object, and does not have the function `trigger`.  This is causing log outs to fail.  Suggest changing back to original code, which used the vanilla `form.submit()` method.